test/support/assertions.rb

```diff
@@ -14,13 +14,13 @@ def assert_line_text(expected)
         result = @last_backlog[2..].join
         expected = Regexp.escape(expected) if expected.is_a?(String)
         msg = "Expected to include `#{expected}` in\n(\n#{result})\n"
-        assert_block(MessageCreationController.new { create_message(msg) }) { result.match? expected }
+        assert_block(ComposeMessageWhenFailed.new { create_message(msg) }) { result.match? expected }
       })
     end
   end
 end
 
-class MessageCreationController
+class ComposeMessageWhenFailed
```

Comment:
imo class name should be a "thing" instead of an "action". for example: `AssertionMessageComposer`.

Reply:
Oh, I see... 🤔 Thank you!

Reply:
Let me put it this way: if `ComposeMessageWhenFailed` can be performed directly, like `ComposeMessageWhenFailed.perform`, I think it's a proper name. Many Rails users do name their job classes this way because the API is `SomeJob.perform_later`.

But in this case, we need to initialize an instance of the class, naming it after an action (verb) is semantically wrong. Because a verb (compose), can not have instances.

(I hope you can understand it. I'm not a native speaker so it's hard for me to explain this concept.)

Reply:
Yeah, maybe It does make sense to me. If this class is used as a class method, the name of class can be verb, but if this class is used as an instance, it should be not verb. Did I get that right?

Reply:
@ono-max exactly 😄